src/adaXT/predict/predict.pyx

```diff
@@ -214,8 +221,16 @@ cdef class PredictLinearRegression(PredictRegression):
                     cur_node = cur_node.left_child
                 else:
                     cur_node = cur_node.right_child
-            Y[i] = cur_node.theta0 + cur_node.theta1*X[i, 0]
-        return Y
+            ind = 0
+            for oo in order:
+                if oo == 0:
+                    deriv_mat[i, ind] = cur_node.theta0 + cur_node.theta1*X[i, 0] + cur_node.theta2*X[i, 0] ** 2
+                if oo == 1:
```

Comment:
should be changed to elif statements, cant both be 0 and 1.

Reply:
done